_posts/2024-12-04-enable-pluggable-storage-in-opensearch-vectordb.md

```diff
@@ -0,0 +1,255 @@
+---
+layout: post
+title:  "Deep dive: Enabling pluggable storage in the OpenSearch vector engine"
+authors:
+ - kdooyong
+ - navneev
+ - vamshin
+ - kolchfa
+date: 2024-12-16
```

Comment:
Update date to 2024-12-18 and we'll get this out tomorrow morning.

Reply:
Updated the date to 2024-12-18 in the article!